mappings/alpha2-to-formats.json

```diff
@@ -180,14 +180,14 @@
     "FK"
   ],
   "GB.json": [
-    "GB",
-    "GG",
-    "IM",
-    "JE"
+    "GB"
   ],
   "GF.json": [
     "GF"
   ],
+  "GG.json": [
```

Comment:
You need to add these new mappings to [formats-web.js](https://github.com/Cimpress-MCP/postal-codes-js/blob/master/formats-web.js) file as well. Unit tests are currently failing.

Reply:
New formats have been added to that file.